{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "607a6d30_6395361c",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1000191
      },
      "writtenOn": "2021-03-01T17:39:19Z",
      "side": 1,
      "message": "So far the cellular_enable_and_connect (similar to the \"safety dance\" autotest) is the only test that seems generally flaky.\n",
      "revId": "c0d54761138a349bfaf353cca9adc308e81db9c3",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}